### PR TITLE
Added /pizza page

### PR DESCRIPTION
**Pull request by DCoxshall**
Added pizza page, which currently displays some recent data handed to me over discord. Currently features two graphs, a bar chart and a pie chart. 
This PR is an attempt to solve [this](https://github.com/CSSUoB/cssuob.github.io/issues/329) issue. Unfortunately, I couldn't find a way link data from any easily-updatable source (i.e. Google Sheets) as was mentioned in the original issue. If this is something that needs to be done, then it's out of my ability range, for now.
Thanks for letting me take a stab at this. I know it's not much, but it's my first PR on a production codebase.

### CODE REVIEW

**Comment by MattyTheHacker** [comment]
The light text on the dark background is a little hard to read
image

other than that it looks pretty good to me. 

I'd also note that when hovering over the pie chart it only shows the number, not what the category is so might be nice to have a key saying which colour is which and/or changing it so hovering over it shows which flavour it is

File: pizzas.html
Comment (LMBishop) [inline pizzas.html:61]: The colours look a bit muted compared to the rest of CSS' colours, though it's not really a massive issue

File: pizzas.html
Comment (LMBishop) [inline pizzas.html:43]: It is worth adding `aria-label` and `role` attributes here (or something similar) for accessibility. See: https://www.chartjs.org/docs/latest/general/accessibility.html

For example `role="img" aria-label="Vegan Vegi Supreme: X, etc."`, or just saying it's a bar chart instead.

**Review by LMBishop — COMMENTED** [review]
Looks good :)

Just a couple of comments below, and one other thing: could it be renamed to `pizza`? Better yet, have `pizzas` redirect to `pizza` too

**Comment by DCoxshall** [comment]
> The light text on the dark background is a little hard to read image
> 
> other than that it looks pretty good to me.
> 
> I'd also note that when hovering over the pie chart it only shows the number, not what the category is so might be nice to have a key saying which colour is which and/or changing it so hovering over it shows which flavour it is

**Comment by DCoxshall** [comment]
wait i closed it by accident

**Comment by DCoxshall** [comment]
Ok so I fixed everything that people mentioned:
- I've turned the labels on the chart axes white
- I've added ARIA text to both charts
- I've made the colours bolder
- I've renamed the new page to /pizza

**Comment by jedevc** [comment]
:eyes: I don't think we need the `package.json` and `package-lock.json` files committed? The page seems to pull by CDN (which is the way to go for this imo).

`node_modules` should definitely *never* be committed.

**Comment by DCoxshall** [comment]
Okay I'm currently learning what "rebasing" is, I will fix this, just give me a minute.

File: css/pizzas.css
Comment (jedevc) [inline css/pizzas.css:14]: I think we probably want to have a `@media` query somewhere here in this file, so that on mobile devices we can change the `flex-direction` to `column`, so that the pie chart shows below the bar chart on mobile, instead of disappearing off the edge of the screen.

**Comment by DCoxshall** [comment]
Hey if this is good to go could someone review it please? If there's anything that needs changing lmk.

**Comment by MattyTheHacker** [comment]
The pie chart still goes off the edge of the screen rather than moving below

**Comment by DCoxshall** [comment]
Ok, how's this? I think I've finally got it fixed.

**Comment by MattyTheHacker** [comment]
Looks good! there's a conflict with the redirects file since we changed it since you forked but if you can resolve that conflict I think this is god

File: _data/redirects.yaml
Comment (MattyTheHacker) [inline _data/redirects.yaml:80]: You've left in the conflict markers here and at the end of the file

Comment (JTWWilson) [reply _data/redirects.yaml:80]: If everything else is good you can just cut that out when merging

**Review by MattyTheHacker — APPROVED** [review]
okay yep this looks good, I'll wait for another review before merging just to be sure but yeah looks good

**Review by JTWWilson — APPROVED** [review]
Looks good, greta job @DCoxshall!

**Pull request merged**
